mod/wikirate/set/type_plus_right/metric/all_values.rb

```diff
@@ -0,0 +1,231 @@
+include Card::CachedCount
+include Type::SearchType
+
+def virtual?; true end
+
+def raw_content
+  %{
+    {
+      "left":{
+        "type":"metric_value",
+        "left":{
+          "left":"_left"
+        }
+      },
+      "right":"value",
+      "limit":0
+    }
+  }
+end
+
+def get_params key, default
+  if (value = Env.params[key])
+    value.to_i
+  else
+    default
+  end
+end
+
+def query params={}
+  default_query = params.delete :default_query
+  @query = super params
+  if !default_query
+    @query[:limit] = params[:default_limit] || 20
+    @query[:offset] = get_params("offset", 0)
+  end
+  @query
+end
+
+def get_cached_result
+  @cached_metric_values ||= begin
+    cached_json = fetch(trait: :cached_count, new: {}).format.render_core
+    JSON.parse(cached_json)
+  end
+end
```

Comment:
Wondering if we shouldn't generalize this the same way we generalize CachedCount.  (Perhaps call it CachedSearchResult)?